src/save_win.rs

```diff
@@ -16,10 +16,37 @@ pub fn load(key: &str) -> Option<String> {
 }
 
 pub fn save(key: &str, value: &str) {
-    let db = Connection::open("save.db").unwrap();
+    let db = get_connection();
     db.execute(
         "REPLACE INTO save (key, value) VALUES (?1, ?2)",
         params![key, value],
     )
     .unwrap();
 }
+
+fn migrate_db(db: &Connection) {
+    let mut stmt = db.prepare("CREATE TABLE IF NOT EXISTS save (key VARCHAR(255) PRIMARY KEY, value LONGTEXT);").unwrap();
+    stmt.execute(params![]).unwrap();
+}
```

Comment:
There's also the `rusqlite_migration` crate that perfectly does the trick with proper SQL-files migrations, but I didn't want to complexify everything right now, the project is perfectly fine as-is :)